pkg/quick/errorutil.go

```diff
@@ -62,17 +62,15 @@ func FormatJSONSyntaxError(data io.Reader, offset int64) (highlight string) {
 		if readBytes > offset {
 			break
 		}
-		switch b {
-		case '\n':
+		if b == '\n' {
```

Comment:
We should add 'continue' after 
```
	readLine.Reset()
	errLine++
```

because we should not add the new line in the json error